skimage/feature/blob.py

```diff
@@ -411,3 +411,136 @@ def blob_doh(image, min_sigma=1, max_sigma=30, num_sigma=10, threshold=0.01,
+
+    Returns
+    -------
+    A : (n, 3) ndarray
+        A 2d array with each row representing 3 values, ``(y,x,sigma)``
+        where ``(y,x)`` are coordinates of the blob and ``sigma`` is the
+        standard deviation of the Gaussian kernel of the Hessian Matrix whose
+        determinant detected the blob.
+
+    References
+    ----------
+    .. [1] http://en.wikipedia.org/wiki/Blob_detection
+
+    .. [2] Herbert Bay, Andreas Ess, Tinne Tuytelaars, Luc Van Gool,
+           "SURF: Speeded Up Robust Features"
+           ftp://ftp.vision.ee.ethz.ch/publications/articles/eth_biwi_00517.pdf
+
+    Examples
+    --------
+    >>> from skimage import data, feature
+    >>> img = data.coins()
+    >>> feature.blob_doh_log(img)
+    array([[123, 205,  20],
+           [124, 336,  20],
+           [126, 101,  20],
+           [126, 153,  20],
+           [197,  44,  20],
+           [197, 153,  20],
+           [123,  44,  23],
+           [192, 212,  23],
+           [193, 275,  23],
+           [195, 100,  23],
+           [262, 243,  23],
+           [265, 113,  23],
+           [121, 271,  30],
+           [185, 348,  30],
+           [260, 173,  30],
+           [270, 363,  30]])
+
+
+    Notes
+    -----
+    The radius of each blob is approximately `sigma`.
+    Computation of Determinant of Hessians is independent of the standard
+    deviation. Therefore detecting larger blobs won't take more time. In
+    methods line :py:meth:`blob_dog` and :py:meth:`blob_log` the computation
+    of Gaussians for larger `sigma` takes more time. The downside is that
+    this method can't be used for detecting blobs of radius less than `3px`
+    due to the box filters used in the approximation of Hessian Determinant.
+    The Hessian-Laplace method sometimes gives better results than the method
+    based on the determinant of Hessian. For example, on the test coin data
+    Hessian-Laplace shows better results compared to determinant of Hessian.
+    (the last method has one false positive while Hessian-Laplace doesn't)
+    """
+    
+    image = img_as_float(image)
+    image = integral_image(image)
+    
+    if log_scale:
+        start, stop = log(min_sigma, 10), log(max_sigma, 10)
+        sigma_list = np.logspace(start, stop, num_sigma)
+    else:
+        sigma_list = np.linspace(min_sigma, max_sigma, num_sigma)
+    
+    # Get approximate determinants of Hessian and Laplacian of Gaussians
+    # Approximations are already scale normalised
+    hessian_and_laplacian_images = [_hessian_matrix_det_and_laplacian(image, s) for s in sigma_list]
+    # Separate hessian and laplacian images
+    hessian_images, laplacian_images = zip(*hessian_and_laplacian_images)
+    
+    laplacian_cube = np.dstack(laplacian_images)
+    
+    # Find local maximas for each hessian image
+    # This points will be checked for Laplacian scal local maximum
+    list_of_peaks = []
+    
+    for scale_number, current_hessian_image in enumerate(hessian_images):
```

Comment:
Hello, @vighneshbirodkar 

Thank you for your review :+1: 

I once implemented a Harris-Laplace.
It is really similar to Hessian-Laplace, but you use Harris measure for finding local maximas
on each scale.
Here is a popular matlab's code for it:
http://www.mathworks.com/matlabcentral/fileexchange/17894-keypoint-extraction/content//keypointExtraction/kp_harrislaplace.m
You can see that they do it for each scale and then test selected points for local maxima in
Laplacian scale space.
And on the wikipedia's page it's said the same:
http://en.wikipedia.org/wiki/Blob_detection#The_Laplacian_of_Gaussian

Did I get you question right?